action.php

```diff
@@ -144,9 +145,39 @@ public function handleHtmlEditformOutput($event, $param) {
         $origtext = io_readWikiPage($file,$id);
 
         // Insert original page on the side
-        $form = & $event->data;
-        $pos = $form->findElementByType('wikitext');
-        if ($pos===false) return;
+        $form = $event->data;
+        if (is_a($form, \dokuwiki\Form\Form::class)) {
+            $pos = $form->findPositionByType('textarea'); // new
+            if ($pos===false) return;
+            $this->handleHtmlEditformOutputNG($form, $pos, $origtext);
+        } else {
+            $pos = $form->findElementByType('wikitext'); // old
```

Comment:
Here too. (See other comment regarding "old" and "new".)

Reply:
done too.